src/main/webapp/app/overview/course-exercises/course-exercise-row.component.ts

```diff
@@ -78,7 +78,7 @@ export class CourseExerciseRowComponent implements OnInit, OnDestroy, OnChanges
 
     ngOnChanges(): void {
         const cachedParticipations = this.participationWebsocketService.getParticipationsForExercise(this.exercise.id!);
-        if (cachedParticipations) {
+        if (cachedParticipations?.length > 0) {
```

Comment:
This should also work.

```suggestion
        if (cachedParticipations?.length) {
```